src/types/publication.types.ts

```diff
@@ -88,18 +93,66 @@ interface DoiRequest {
   status: DoiRequestStatus;
 }
 
+export interface JournalPublication extends BackendType, PublicationFileSet {
+  readonly identifier: string;
+  readonly createdDate: string;
+  readonly owner: string;
+  readonly status: PublicationStatus;
+  readonly doiRequest: DoiRequest | null;
+  doiRequested: boolean;
+  entityDescription: JournalEntityDescription;
+  project: Project | null;
+}
+
+export interface DegreePublication extends BackendType, PublicationFileSet {
+  readonly identifier: string;
+  readonly createdDate: string;
+  readonly owner: string;
+  readonly status: PublicationStatus;
+  readonly doiRequest: DoiRequest | null;
+  doiRequested: boolean;
+  entityDescription: DegreeEntityDescription;
+  project: Project | null;
+}
+
+export interface BookPublication extends BackendType, PublicationFileSet {
+  readonly identifier: string;
+  readonly createdDate: string;
+  readonly owner: string;
+  readonly status: PublicationStatus;
+  readonly doiRequest: DoiRequest | null;
+  doiRequested: boolean;
+  entityDescription: BookEntityDescription;
+  project: Project | null;
+}
+
+export interface ReportPublication extends BackendType, PublicationFileSet {
+  readonly identifier: string;
+  readonly createdDate: string;
+  readonly owner: string;
+  readonly status: PublicationStatus;
+  readonly doiRequest: DoiRequest | null;
+  doiRequested: boolean;
+  entityDescription: ReportEntityDescription;
+  project: Project | null;
+}
```

Comment:
Alle disse ser like ut for meg..?

Reply:
alt bortsett fra entityDescription :) :) :)

Reply:
aha, ofc :D 
kan vi legge resten i en `BasePublication` som extendes elns?

Reply:
Ja, det var lurt! 😁